stakingModel/test/SimpleDAIStaking.e2e.test.js

```diff
@@ -0,0 +1,72 @@
+const SimpleDAIStaking = artifacts.require("SimpleDAIStaking");
+const GoodDollar = artifacts.require("GoodDollar");
+const DAIMock = artifacts.require("DAIMock");
+const cDAIMock = artifacts.require("cDAIMock");
+
+const fse = require("fs-extra");
+
+const BN = web3.utils.BN;
+export const BLOCK_INTERVAL = 2;
+export const NULL_ADDRESS = "0x0000000000000000000000000000000000000000";
+export const DAI_ADDRESS  = "0x4F96Fe3b7A6Cf9725f59d353F723c1bDb64CA6Aa";
+export const cDAI_ADDRESS = "0xe7bc397dbd069fc7d0109c0636d06888bb50668c";
+
+contract("SimpleDAIStaking - staking with DAI mocks", ([founder, staker]) => {
```

Comment:
add network to parameters and dont run if not on correct network (ie if network is test/develop)

Reply:
?

Reply:
waiting

Reply:
skipping the test if the network is test or develop